.ansible-lint

```diff
@@ -1,6 +1,11 @@
 skip_list:
    - '102'  # No Jinja2 in when
+   - '106'  # Role name {} does not match ``^[a-z][a-z0-9_]+$`` pattern
+   - '208'  # File permissions not mentioned
+   - '301'  # Commands should not change things if nothing needs doing
    - '303'  # systemctl used in place of systemd module
-   - '502'  # All tasks should be named
+   - '306'  # Shells that use pipes should set the pipefail option 
    - '602'  # Don't compare to empty string
    - '701'  # Role info should contain platforms
+warn_list:
+   - '502'  # All tasks should be named
```

Comment:
I know this is fairly minor, but it gives me pause to see this list growing and growing. 

* 106 I understand because it conflicts with our other conventions.
* 208 Is suspect - it should be easy to just specify the permissions.
* 502 is the same - why not follow the convention and name the tasks?
* 301 and 306 are a bit concerning, maybe I'll see them as a continue down the review

Reply:
208 is a TODO
502 is also TODO, I need to go back and name all tasks at some point, there are quite a few blocks or debugs.
301 is there because it is flagged every time shell or command is used to get some information ( I think I have to add a changed_when option)
306 is new because it recently started getting flagged.